Crashlytics/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Unreleased
+- [changed] Added support for [crashlytics] to report information from [remote_config].
```

Comment:
@morganchen12, does your tooling pick up process `[crashlytics]` and `[remote_config]` or should they instead be expressed as Crashlytics and Remote Config?

Reply:
Checked codebase I don't see anywhere use product tag, changed to name instead. Thank you for the checking!